sdk/core/azure-core/inc/azure/core/internal/extendable_enumeration.hpp

```diff
@@ -20,7 +20,7 @@ namespace Azure { namespace Core { namespace _internal {
    * Example:
    *
    * \code{.cpp}
-   * class MyEnumeration final : public ExtendableEnumeration<AttestationType> {
+   * class MyEnumeration final : public ExtendableEnumeration<MyEnumeration> {
    * public:
    *   explicit MyEnumeration(std::string attestationType) :
```

Comment:
Should we generalize other instances in the doc comments that are specific to just attestation, like this (and below), @LarryOsterman?

Maybe like `serviceEnumType`

Reply:
Agreed. THis was a sloppy C&P on my part. @Jinming-Hu would you mind doing that as well as a part of this pull request.